src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -64,10 +65,20 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                     this.WriteLine($"{"LogRecord.Exception:".PadRight(RightPaddingLength)}{logRecord.Exception?.Message}");
                 }
 
-                logRecord.ForEachScope(ProcessScope, this);
+                logRecord.ForEachScope(ProcessScopeItem, this);
 
-                static void ProcessScope(object scope, ConsoleLogRecordExporter exporter)
-                    => exporter.WriteLine($"{"LogRecord.Scope:".PadRight(RightPaddingLength)}{scope}");
+                static void ProcessScopeItem(LogRecordScope scope, ConsoleLogRecordExporter exporter)
```

Comment:
Curious: which one is more "native" to .NET - `ProcessScopeItem` or `ProcessScopeItems` (plural form)?

Reply:
I think the singular here is fine because it is getting called once for each scope and not being passed a list/enumerable of scopes? I did rename it back to "ProcessScope" (instead of "ProcessScopeItem"). That was a carry-over from me messing with the API.

Let me know if you want it plural, I don't feel strongly about it.

Reply:
I don't have strong preference here, it was just coming from curiosity.